x-pack/docs/en/rest-api/security.asciidoc

```diff
@@ -124,12 +124,12 @@ realm when using a custom web application other than Kibana
 [[security-enrollment-apis]]
 === Enrollment
 
-You can use the following APIs to allow new nodes to join an existing cluster with
-security enabled or to allow a client to configure itself to communicate with
-a secured {es} cluster
+Use the following APIs to enable new nodes to join an existing cluster with
+security enabled, or to enable a client to configure itself to communicate with
```

Comment:
```suggestion
security enabled, or to enable a kibana instance to configure itself to communicate with
```

can we add this change that I missed in the original PR too please?

Reply:
+1 -- I just pushed a change in 95545bc214cc2b5de42d47a40db41ea112931680